Bobsar0/Fast-Food-Fast
# #161570901 Manage menu items

#### What does this PR do?
Manage food items on the menu

#### Description of Task to be completed?
- Create and consume `PUT /menu/:foodid` endpoint so admin can edit food items
- Create and consume `DELETE /menu/:foodid` endpoint so admin can delete food items

#### How should this be manually tested?
- Clone the repo and run the following commands:
```
  - git checkout ft-manage-menu-items-#161570901  
  - npm install
  - npm start
```
- Navigate to http://localhost:9999/ and login/create an account using the credentials below
``` 
 username: admin
 email: admin@fastfoodfast.com
 password: admin!23
```
_You should be redirected to a dedicated menu page_
- Navigate to `http://localhost:9999/templates/admin.html`
- Click on **Administrator Panel** > **Manage Menu items**
_You should see a table containing the details of all food items on the menu_
- Use the search feature at the top of the table to search for a food item.
- Use the edit and delete items buttons to manage a food item

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/#161570901 

#### Screenshots
![image](https://user-images.githubusercontent.com/19636985/48076031-b0373000-e1e4-11e8-9fae-a07f68d6b99a.png)

![image](https://user-images.githubusercontent.com/19636985/48076106-d230b280-e1e4-11e8-8f56-f9fa1c62e216.png)

![image](https://user-images.githubusercontent.com/19636985/48076143-e5438280-e1e4-11e8-8edb-f8959c0ab512.png)


![image](https://user-images.githubusercontent.com/19636985/48075879-63535980-e1e4-11e8-96f0-8548c2273f3e.png)
